idea_town/frontend/static-src/app/templates/account-inactive-page.js

```diff
@@ -1,11 +1,16 @@
 export default `
-  <div class="page">
-    <section class="page main-content" data-hook="account-inactive-page">
-      <header data-hook="main-header"></header>
-      <h1>Account Inactive</h1>
-      <h2>We're not quite ready for you yet!<br>
-          Please come back in a few weeks.</h2>
+  <div class="page" id="full-page-notice">
+    <section class="main-content" data-hook="account-inactive-page">
+      <div class="vertical-centering-container">
+        <div id="beta-notice-modal" class="delayed-fade-in">
+          <h1>Thanks for signing up!</h1>
+          <h2>We'll let you know as soon as Idea Town is ready.</h2>
```

Comment:
What does _"We'll let you know"_ mean? Are we collecting email addresses somewhere?
How will we notify these random people?